assets/src/sass/base/_base.scss

```diff
@@ -1,17 +1,74 @@
-/*--------------------------------------------------------------
-# Base styles
---------------------------------------------------------------*/
-
-/* Reset
---------------------------------------------- */
-@import "resets";
-
-/* Typography
---------------------------------------------- */
-@import "typography/headings";
-@import "typography/typography";
-
-/* Elements
---------------------------------------------- */
-@import "elements";
-@import "forms";
+/*--------------------------------------------------------------
+# Base styles
+--------------------------------------------------------------*/
+
+/* Reset
+--------------------------------------------- */
+@import "resets";
+
+/* Typography
+--------------------------------------------- */
+@import "typography/headings";
+@import "typography/typography";
+
+/* Elements
+--------------------------------------------- */
+@import "elements";
+@import "forms";
+
+/* Layout (block style)
+--------------------------------------------- */
+:root {
+	--wp--style--global--content-size: 980px;
+	--wp--style--global--wide-size: 1020px;
+}
+.single-post {
+	--wp--style--global--content-size: 40rem;
+	--wp--style--global--wide-size: 980px;
+}
+
+@media screen and (max-width: #{$width-breakpoint-desktop}) {
+	:root {
+		--wp--style--global--content-size: calc( 100vw - 5rem );
+		--wp--style--global--wide-size: calc( 100vw - 5rem );
+	}
+	.single-post {
+		--wp--style--global--content-size: calc( 100vw - 5rem );
+		--wp--style--global--wide-size: calc( 100vw - 5rem );
+	}
+}
+
+@media screen and (max-width: #{$width-breakpoint-tablet}) {
+	:root {
+		--wp--style--global--content-size: calc( 100vw - 40px );
+		--wp--style--global--wide-size: calc( 100vw - 40px );
+	}
+	.single-post {
+		--wp--style--global--content-size: calc( 100vw - 40px );
+		--wp--style--global--wide-size: calc( 100vw - 40px );
+	}
+}
```

Comment:
There may well be a better way to handle this, but using media rules to alter these variables was the simplest way I could achieve today's padding behavior without actually adding complex padding rules.